src/classes.h

```diff
@@ -13,5 +15,7 @@ namespace {
         std::pair<std::string, bool> dummy6;
```

Comment:
Question: why do we have these dummy# names? Can't we redefine them in a more meaning-oriented way?

Reply:
they are only there to force the type declaration for the dictionary to pick up, nothing else should include classes.h (I vaguely remember seeing this convention being followed elsewhere as well, but as long as they are unique anything goes)